packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -845,24 +1027,23 @@ class IOSDeviceLogReader extends DeviceLogReader {
   /// is true.
   final List<String> _streamFlutterMessages = <String>[];
 
-  /// When using both `idevicesyslog` and `ios-deploy`, exclude logs with the
-  /// "flutter:" prefix if they have already been added to the stream. This is
-  /// to prevent duplicates from being printed.
-  ///
-  /// If a message does not have the prefix, exclude it if the message's
-  /// source is `idevicesyslog`. This is done because `ios-deploy` and
-  /// `idevicesyslog` often have different prefixes on non-flutter messages
-  /// and are often not critical for CI tests.
+  /// When using multiple logging sources, exclude logs with a `flutter:` prefix
+  /// if they have already been added to the stream. This is to prevent duplicates
+  /// from being printed.
   bool _excludeLog(String message, IOSDeviceLogSource source) {
-    if (!useBothLogDeviceReaders) {
+    if (!usingMultipleLoggingSources) {
```

Comment:
could you explain why the naming change?

Reply:
Before it was only using 2 at a time, so "both" made more sense. With new code, it could be up to 3 at a time, so I changed it to "multiple" and I just thought "LoggingSources" made more sense that "LogDeviceReaders".